packages/forklift-console-plugin/src/modules/Providers/views/details/tabs/VirtualMachines/utils/helpers/getHighestPriorityConcern.ts

```diff
@@ -1,21 +1,24 @@
 import { ProviderVirtualMachine } from '@kubev2v/types';
 
+import { hasConcerns } from './hasConcerns';
+
 type ConcernCategory = 'Critical' | 'Warning' | 'Information';
 
 export const getHighestPriorityConcern = (vm: ProviderVirtualMachine): ConcernCategory => {
-  if (!vm?.concerns) {
+  const concerns = hasConcerns(vm) ? vm.concerns : [];
```

Comment:
```suggestion
  const concerns = vm?.concerns || []; 
```

if it does not have concerns it will be []

Reply:
![image](https://github.com/kubev2v/forklift-console-plugin/assets/64194103/9fd646ca-b020-4f37-9e0b-c76a3789102e)

Reply:
then add the missing property ?

Reply:
I've just removed it:) the purpose of this PR is to adjust typing to the one used by backend code.
Note however that this problem will show up in any renderer that will try to access a property not present in all provider specific types (most of the props)

Reply:
try adding a type property like proposed in https://github.com/kubev2v/forklift-console-plugin/pull/723#discussion_r1328376203

Reply:
`providerType` property will be added in a separate PR